src/index.ts

```diff
@@ -5,3 +5,4 @@ export { default as Geometric } from "./geometric";
 export { default as Power } from "./power";
 export { default as Prime } from "./prime";
 export { default as Triangle } from "./triangle";
+export { default as Twin } from "./twin";
```

Comment:
FYI: I added this earlier - this is basically the main entry point for people importing this library , so if you add new sequences then you need to add a line to this file to make them available outside.